kiosk/src/Transforms/exitToEnterHighScore.ts

```diff
@@ -20,11 +20,12 @@ export function exitToEnterHighScore(): void {
     launchedGameHighs = launchedGameHighs || [];
     const currentHighScore = state.mostRecentScores[0];
```

Comment:
Is this getting the best score? Would you want to be getting the third-place score (or however many places are listed)?

Reply:
This is getting the score that was sent from the simulator from the state, so it doesn't know what place the score will go yet.